rclcpp/include/rclcpp/duration.hpp

```diff
@@ -44,69 +55,98 @@ class RCLCPP_PUBLIC Duration
   // cppcheck-suppress noExplicitConstructor
   Duration(const builtin_interfaces::msg::Duration & duration_msg);  // NOLINT(runtime/explicit)
 
+  /// Time constructor
+  /**
+   * \param duration rcl_duration_t structure to copy.
+   */
   explicit Duration(const rcl_duration_t & duration);
 
+  /// Copy constructor.
   Duration(const Duration & rhs);
 
+  /// Duration destructor.
   virtual ~Duration() = default;
 
+  /// Return a builtin_interfaces::msg::Duration object based
   operator builtin_interfaces::msg::Duration() const;
 
   // cppcheck-suppress operatorEq // this is a false positive from cppcheck
   Duration &
   operator=(const Duration & rhs);
 
+  /// Copy assign operator from a builtin_interfaces::msg::Duration.
```

Comment:
I don't think the operator comments below necessary? You're not really saying anything informative